mlos_bench/mlos_bench/tests/optimizers/mock_opt_test.py

```diff
@@ -47,6 +41,21 @@ def mock_configurations() -> list:
     ]
 
 
+@pytest.fixture
+def mock_configurations(mock_configurations_no_defaults: list) -> list:
+    """
+    A list of 2-tuples of (tunable_values, score) to test the optimizers.
+    """
+    return [
+        ({
+            "vmSize": "Standard_B4ms",
+            "idle": "halt",
+            "kernel_sched_migration_cost_ns": -1,
+            "kernel_sched_latency_ns": 2000000,
+        }, 88.88),
```

Comment:
Any way to pull the defaults out of the actual optimizer.  Worries me that that's defined elsewhere and there will be unrelated files to edit when one of those changes, but maybe no way to avoid it.

Reply:
Technically, we can extract them from the `tunable_groups` fixture, but would rather keep this code as is